modules/videopress/videopress.php

```diff
@@ -56,24 +56,47 @@ function on_init() {
 
 		if ( $this->can( 'upload_videos' ) && $options['blog_id'] ) {
 			add_action( 'wp_ajax_videopress-get-upload-token', array( $this, 'wp_ajax_videopress_get_upload_token' ) );
+			add_filter( 'plupload_default_settings', array( $this, 'videopress_pluploder_config' ) );
 		}
 
 		add_filter( 'videopress_shortcode_options', array( $this, 'videopress_shortcode_options' ) );
 	}
 
+	/**
+	 * Ajax method that is used by the VideoPress uploader to get a token to upload a file to the wpcom api.
+	 *
+	 * @return void
+	 */
 	function wp_ajax_videopress_get_upload_token() {
-		if ( ! $this->can( 'upload_videos' ) )
-			return wp_send_json_error();
+		if ( ! $this->can( 'upload_videos' ) ) {
+			wp_send_json_error();
+			return;
```

Comment:
We can probably skip the `return;` here -- it's fine to leave in for clarity's sake, but it will never run.  `wp_send_json()` -- which is called by `wp_send_json_error()` -- ends with either a `die;` or `wp_die();` depending on if it's an ajax request according to `DOING_AJAX`.